### DISCUSSION TITLE
AI medics still mishandle fentanyl and kill crew via it

### DISCUSSION THREAD

**Discussion by PanmanS** [FakeFishGames/Barotrauma#13688] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Well, bots still heal you at extremely low HP loss with fentanyl causing instant overdose
https://youtu.be/CUN8w2zd290

### Reproduction steps

1, spawn bot, get damaged, give him fentanyl
2. enjoy the show

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.2.13.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I gave this a quick test, and could not find a combination of afflictions that would've caused a bot to overdose a character. Tested 10, 15, 20 and 25 internal damage: the bots refused the treat the lowest levels at all, and wouldn't use fentanyl if the character was already suffering of opiate addiction or overdose.

If you have an idea of what kind of a situation this occurred in (what kind of afflictions were the bots treating?) I'm having a hard time figuring out what's happening in the video: the character seems to have such a minuscule amount of damage, I don't understand why the bots are treating the character in the first place (in my testing, they just say "xxxx's injuries are so minor I don't think I should waste any meds on them"). I also don't see any medic bots in the video. So if you can give any more pointers on this, it would most likely help us a lot in diagnosing this: what are the orders on the bot(s), what kind of a bot is it, what are the afflictions...?

**Reply by PanmanS** [reply]
Tyler is medic bot with order to heal targets, he had 2 fentanyl in his inventory and nothing else (other meds were on me and in medcab of dugong)
And he also sends messages in chat.

Medicine is entirely vanilla btw

**Reply by PanmanS** [reply]
He also commented need in opiates.

**Reply by PanmanS** [reply]
[Save.zip](https://github.com/FakeFishGames/Barotrauma/files/14794403/Save.zip)
It is modded save with that issue, but medicine is unchanged, so should work fine

**Comment by Regalis11** [comment]
It seems the crew vitality multiplier in the save is set so low, a single syringe of fentanyl is enough to kill a character. Thank you for the extra info, we'll need to look into this!

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/pull/5113 (PR in our private repo)
